src/components/RenderHTML.js

```diff
@@ -113,23 +114,15 @@ function ImgRenderer({tnode}) {
     );
 
     return (
-        <AttachmentModal
-            title="Attachment"
-            sourceURL={source}
-            isAuthTokenRequired={isAttachment}
+        <TouchableOpacity
+            onPress={() => setAttachmentModalData({sourceURL: source, isAttachment, isModalOpen: true})}
```

Comment:
I think this could be greatly improved if we restore the ability to pass props here. 

If we can't then for now, we might want to just stop using the `img` renderer for images. Instead we could parse the html attributes from the `message.html`. We would need some way to tell they are an attachment. Something similar to this:

https://github.com/Expensify/Expensify.cash/blob/b2c03fc5a8bd0d2c5a3f67847d132af1d41702e1/src/libs/actions/Report.js#L230-L237

But for all new actions we are adding here

https://github.com/Expensify/Expensify.cash/blob/b2c03fc5a8bd0d2c5a3f67847d132af1d41702e1/src/libs/actions/Report.js#L405-L412

I'd prefer to just wait until we can add props.

Reply:
Yeah additionally it would also be amazing if we can add sequenceNumber when we call setAttachmentModalData just so that the attachmentModal knows where the current attachment belongs in the reportActions that looping everytime to find report action having the matching source url.